src/routes/llms.txt/+server.ts

```diff
@@ -0,0 +1,160 @@
+import type { RequestHandler } from '@sveltejs/kit';
+
+export const prerender = true;
+
+// Eagerly import all markdown/markdoc content within src/routes
+const markdocAndMarkdownFiles = import.meta.glob('$routes/**/*.{markdoc,md}', {
+    query: '?raw',
+    import: 'default',
+    eager: true
+});
+
+// Strip group directories like (marketing) from a route path
+function stripRouteGroups(routePath: string): string {
+    return routePath.replace(/\([^/]+\)\//g, '');
+}
+
+function pathToRoute(path: string): string {
+    // Convert absolute file path to a route path
+    // Examples:
+    //  /src/routes/docs/intro/+page.markdoc => /docs/intro
+    //  /src/routes/blog/post/my-post.markdoc => /blog/post/my-post
+    //  /src/routes/docs/references/xyz/page.md => /docs/references/xyz/page
+    let route = path.replace(/^\/src\/routes/, '');
+
+    if (/\/+page\.(markdoc|md)$/.test(route)) {
+        route = route.replace(/\/+page\.(markdoc|md)$/, '');
+    } else {
+        route = route.replace(/\.(markdoc|md)$/, '');
+    }
+
+    // Remove route groups
+    route = stripRouteGroups(route.replace(/\\/g, '/'));
+
+    // Also handle extension-less +page keys just in case
+    route = route.replace(/\/+page$/, '');
+
+    return route || '/';
```

Comment:
+page are still in the URLs:

<img width="691" height="297" alt="image" src="https://github.com/user-attachments/assets/6d2b23a9-8a77-408e-b95f-706912bea09c" />